src/constants.js

```diff
@@ -3,7 +3,7 @@ import os from 'os'
 
 import { version } from '../package.json'
 
-export const DEFAULT_SAUCE_CONNECT_VERSION = '4.7.1'
+export const DEFAULT_SAUCE_CONNECT_VERSION = '4.8.0-rc2'
```

Comment:
as mentioned elsewhere, we probably don't want to upgrade to prerelease versions unless we offer a prerelease version of _this_ package... so not a dealbreaker necessarily.  What's the reasoning behind upgrading to a prerelease (and not waiting for v4.8.0)?

Reply:
The MR is WIP, I'll wait for the full release before merging it